test/plain-list/AutosuggestApp.js

```diff
@@ -54,6 +54,11 @@ export const onSuggestionsUpdateRequested = sinon.spy(({ value }) => {
   });
 });
 
+let selectFirstSuggestionValue = false;
+export function setSelectFirstSuggestion(value){
```

Comment:
Did you run `npm run lint`? I would expect ESLint to complain that a space is missing before `{`.